test/evergreen.yml

```diff
@@ -291,13 +291,19 @@ functions:
         export WIREDTIGER_CONFIG='checkpoint_sync=0,transaction_sync=(method=none)'
         CMD='./t -h WT_TEST.$i.$t -t r -r 2 -W 3 -n 1000000 -k 1000000 -C "cache_size=100MB"'
 
+        ret=-1
         for i in $(seq ${times|1}); do
           for t in $(seq ${no_of_procs|1}); do
             eval nohup $CMD > nohup.out.$i.$t 2>&1 &
           done
 
           for t in $(seq ${no_of_procs|1}); do
-            wait -n || exit $?
+            wait -n || ret=$?
+            if [ $ret -ne -1 ]; then
+              # Skip the below lines from nohup output file.
```

Comment:
I know the reason of skipping those lines is because they are very verbose. It would be nice to state the reason in the comment to help with our future-selves :)

Reply:
Yes, I agree with Luke.

Reply:
Fixed